<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>826</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Setting"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDIP">
    <property name="title">
     <string>DIP</string>
    </property>
    <addaction name="actionOpen_Picture"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionConrtours"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_All"/>
    <addaction name="actionHide_All"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_QT"/>
    <addaction name="actionAbout_App"/>
    <addaction name="actionAbout_GTS_400"/>
   </widget>
   <widget class="QMenu" name="menuMachineLearning">
    <property name="title">
     <string>MachineLearning</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionPrevious_Window"/>
    <addaction name="actionNext_Window"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuView"/>
   <addaction name="menuDIP"/>
   <addaction name="menuMachineLearning"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_Setting">
   <property name="text">
    <string>Save Setting</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen_Picture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open Picture</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionAbout_App">
   <property name="text">
    <string>About App</string>
   </property>
  </action>
  <action name="actionDIP">
   <property name="text">
    <string>DIP</string>
   </property>
  </action>
  <action name="actionMotionControl">
   <property name="text">
    <string>MotionControl</string>
   </property>
  </action>
  <action name="actionAxis_Stat">
   <property name="text">
    <string>Axis Stat</string>
   </property>
  </action>
  <action name="actionAxis_Setting">
   <property name="text">
    <string>Axis Setting</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionConrtours">
   <property name="text">
    <string>Conrtours</string>
   </property>
  </action>
  <action name="actionPrevious_Window">
   <property name="text">
    <string>Previous Window</string>
   </property>
  </action>
  <action name="actionNext_Window">
   <property name="text">
    <string>Next Window</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>Simplified Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout_GTS_400">
   <property name="text">
    <string>About GTS-400</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionHide_All">
   <property name="text">
    <string>Hide All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
